src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -41,6 +42,9 @@ export class VerifyApp extends React.Component {
     if (shouldCheckAccount) {
       this.checkAccountAccess();
     }
+    if (!this.props.profile.loading && prevProps.profile.loading) {
+      focusElement('h1');
```

Comment:
We'll need to still add the `tabindex="-1"` onto the H1 for it to take focus. That should happen in the JS though, and also a blur event to remove the attribute.

Here's an example @mpelzsherman put together of a similar behavior: https://github.com/department-of-veterans-affairs/vets-website/commit/2f329a6516f41a51178bd754b22e9bfd8fe3f940

Reply:
The `focusElement` function handles that for us. I confirmed that the Cypress test is legit and this works as intended.

Reply:
eh, looks like that function isn't quite right, but we'll deal with it in another PR. Thanks!

Reply:
I thought about using `focusElement` in my other PR, but it doesn't remove the tabindex on blur.

Reply:
I'm going to submit a PR to update that function

Reply:
Tag me in it so I can take a look. If I'm doing it incorrectly then I'd rather learn the right way